commons/ui/src/main/kotlin/com/teacherworkout/commons/ui/navigation/AppDestinations.kt

```diff
@@ -2,6 +2,25 @@ package com.teacherworkout.commons.ui.navigation
 
 object AppDestinations {
 
+    object Features {
+        /**
+         * The account feature
+         */
+        const val account = "account"
+
+        /**
+         * The home feature
+         */
+        const val home = "home"
+    }
+
+    object Home {
+        /**
+         * The landing screen for home feature
+         */
+        const val landing = "home-landing"
+    }
```

Comment:
What do you think about this separation, I propose we have each `navSubGraph` in it's own class

Reply:
Looks good, the only thing I would change is renaming the _Features_ object to _Routes_ because those are identifiers for routes. Then at the call site we would have:

`navigation(startDestination = AppDestinations.landing, route = AppDestinations.Routes.account)`